### ISSUE TITLE
[Bug] : Unable to remove divider

### ISSUE THREAD

**Issue by yatendra2001** [AppFlowy-IO/appflowy-editor#379]
### Bug Description

I am unable to remove the divider using the keyboard navigation.  Divider only gets removed by using the cursor and then delete button.  

![ezgif.com-video-to-gif\|600x383](https://global.discourse-cdn.com/standard11/uploads/appflowy/original/1X/454762638f93ab4ccaf76ce75311c2958c0464e9.gif)

### How to Reproduce

1. Open example editor
2. Write something
3. Make a divider
4. Then again write something
5. Try to navigate to divider
6. Then press delete or backspace and it won't work.

### Expected Behavior

This is how notion works?

![ezgif.com-video-to-gif (1)\|600x117](https://global.discourse-cdn.com/standard11/uploads/appflowy/original/1X/4621b57734be33723d6949285fcab18e93952993.gif)

### Operating System

MacOS

### AppFlowy Editor Version(s)

main

### Additional Context

Can be related to #370
